cluster/loadbalance/maglevconsistenthashing/maglev.go

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package maglevconsistenthashing
+
+import (
+	"math"
+
+	"github.com/cespare/xxhash/v2"
+
+	"dubbo.apache.org/dubbo-go/v3/protocol"
+)
+
+type Maglev struct {
+	invokers    []protocol.Invoker // Instance
+	lookupTable []int
+
+	tableSize int // Prime number
+}
+
+func NewMaglev(invokers []protocol.Invoker) *Maglev {
+	// Next invokers Prime number
+	pn := NextPrime(len(invokers))
+	m := &Maglev{
+		lookupTable: make([]int, pn),
+		invokers:    invokers,
+		tableSize:   pn,
+	}
+	m.buildLookupTable()
+	return m
+}
+
+// buildLookupTable 构建 Maglev 查找表
```

Comment:
Just curious, why are the annotations in Chinese used here, and other places contain English annotations? Can they all be unified into English annotations?